sdk/python/feast/offline_store.py

```diff
@@ -87,8 +87,14 @@ class OfflineStore(ABC):
     @staticmethod
     @abstractmethod
     def pull_latest_from_table(
-        feature_view: FeatureView, start_date: datetime, end_date: datetime,
-    ) -> Optional[pyarrow.Table]:
+        table_ref: str,
```

Comment:
Is there a reason why we have added all of the fields to the `pull_latest_from_table()` method instead of using `feature_view` which already contains all of the information?

Reply:
My goal is for this function to do just the backend-specific logic, so I don't want to run the field mappings in this function since anyone building a new connector would need to know they have to call the field mapping methods in their implementation of this method. But you're right that table_ref is too tightly coupled to BQ, so I replaced it with the data source, which I think should work